plugins/appInit.ts

```diff
@@ -1,10 +0,0 @@
-import { Plugin } from '@nuxt/types';
-
-const appInitPlugin: Plugin = (context) => {
-  const { serverUsed } = context.store.state.servers;
-  if (serverUsed) {
-    context.$axios.setBaseURL(serverUsed.address);
-  }
-};
-
-export default appInitPlugin;
```

Comment:
Why is this deleted?

Reply:
It was unused, there's an appInitPlugin.ts with the same contents, which was the one being imported in the config.